auto_landing/src/tracking.cpp

```diff
@@ -0,0 +1,50 @@
+#include<auto_landing/tracking.hpp>
+
+
+namespace ariitk::auto_landing {
+
+ void Tracking::init(ros::NodeHandle& nh) {
+
+        set_firefly_pose_ = nh.advertise<geometry_msgs::PoseStamped>("/firefly/command/pose", 1);
+        husky_pose_ = nh.subscribe("/husky_velocity_controller/odom", 1, &Tracking::poseClbk,this);
+        landing_server_ = nh.advertiseService("to_land", &Tracking::serverClbk,this);
+        landing_client_ = nh.serviceClient<std_srvs::Trigger>("to_land");
+
+
+}
+
+void Tracking::run(){
+
+    srv_.response.success=false;
+    
+    
+    
+        if(((husky_odom_.pose.pose.position.x-0.1<=setpt_.pose.position.x)||
```

Comment:
Instead of writing two seperate conditions for less than and greater than, why don't you just use the abs() function?